src/main/java/sirius/web/controller/Route.java

```diff
@@ -212,7 +212,13 @@ private static int compileRouteURI(Route result, String[] elements, StringBuilde
                 Object effectiveValue = Value.of(value).coerce(parameterTypes[idx - 1], null);
                 setAtPosition(result, idx, effectiveValue);
             } else if ("**".equals(expr.getFirst())) {
-                result.add(Arrays.asList(value.split("/")));
+                //we need to split the encoded values so we dont mistake data for the delimiter
+                ArrayList<String> values = new ArrayList<>(Arrays.asList(m.group(i).split("/")));
```

Comment:
Arrays.asStream().map(this::decode).collect(Lambdas.into(result))

und decode als neue methode für value -> URLDecoder.decode(value, Charsets.UTF_8.name()) (das brauchen wir ja noch iwo